spec/acceptance/foreman_cli_plugins_spec.rb

```diff
@@ -26,7 +26,7 @@ class { 'foreman::cli':
 
   it_behaves_like 'hammer'
 
-  ['discovery', 'remote_execution', 'tasks', 'templates'].each do |plugin|
+  ['discovery', 'remote_execution', 'tasks', 'templates', 'katello'].each do |plugin|
```

Comment:
```
expected Package "tfm-rubygem-hammer_cli_foreman_katello" to be installed
```

you'll need to fixup the logic below to not add the `foreman_` prefix and add it to the plugins here, thanks katello.

Reply:
You're missing the actual `include foreman::cli::katello`. I would expect that it's not packaged for Debian so add it like ansible/azure. Right now we also don't have any explicit checks for those and assume that if Puppet worked correctly, it's good enough.